Yosemite/Yosemite/Stores/Order/OrdersUpsertUseCase.swift

```diff
@@ -60,6 +60,7 @@ struct OrdersUpsertUseCase {
         handleOrderTaxes(readOnlyOrder, storageOrder, storage)
         handleOrderCustomFields(readOnlyOrder, storageOrder, storage)
         handleOrderGiftCards(readOnlyOrder, storageOrder, storage)
+        handleOrderAttributionInfo(readOnlyOrder, storageOrder, storage)
```

Comment:
Just wondering since I don't have much context: We do not have any feature flag at the moment that we would use  to wrap this, right?